models/demos/mamba/tt/full_model.py

```diff
@@ -16,6 +16,7 @@
 
 class TtTensorLoader:
     def __init__(self, state_dict, device, tt_cache_path: Optional[str] = None):
+        logger.info(f"Using tensor cache at filepath={tt_cache_path}")
```

Comment:
```
if os.path.exists(tt_cache_path):
    logger.info(f"Using tensor cache at filepath={tt_cache_path}")
```

Reply:
It does not check tensor level cache existence but only the model version folder has created then it could be loading from cache